### ISSUE TITLE
Prerequisites: Change where opening VSC from command line is introduced

### ISSUE THREAD

**Issue by dloomz** [TheOdinProject/curriculum#27941]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

<img width="698" alt="Screenshot 2024-05-07 at 16 27 52" src="https://github.com/TheOdinProject/curriculum/assets/141953494/67405fe3-05e3-4fee-9ff4-89c0a170d671">

After learning how to use the terminal you are given a task. One step is to open the text file created with VSCode, we don't learn this in the external resource so this was a bit confusing. In the end, I did that step via the GUI. As I continued to read I discovered that we are told later how to open VSC and files with VSC from the terminal.

<img width="730" alt="Screenshot 2024-05-07 at 16 30 28" src="https://github.com/TheOdinProject/curriculum/assets/141953494/6b0b03ea-d9ef-4183-93ba-11ed276cb4e4">

Personally, I believe introducing this before the task would make more sense.

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-command-line-basics

### (Optional) Discord Name

dloomzx

**Comment by MaoShizhong** [comment]
Thanks for opening this issue @dloomz. I agree with you, and as per our linting rules (which were implemented after this lesson was written), the assignment section needs to go after the 2 sections that come after it anyway.

Since you didn't check the assignment box, I'll open this up so someone can comment below if they wish to be assigned.

The following changes would need to be made:
- Move the entire `### Assignment` section (lines 49 - 82) to right before the `### Knowledge Check` section, so that it goes `Use the command line like a pro -> Opening files in VSCode from the command line -> Assignment -> Knowledge check`
- Ensure no linting errors due to blank lines. Easiest way to handle this would be to make this change locally, then as per the [repo contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md), run `npm run fix:lesson -- foundations/installations/command_line_basics.md` once, which should fix all remaining lint errors.

**Comment by BlueVelocity** [comment]
I'd be happy to take this on @MaoShizhong .

**Comment by MaoShizhong** [comment]
Assigned, @BlueVelocity :rocket:
